src/render/index.html.eco

```diff
@@ -81,38 +125,40 @@ layout: default
       </div>
     </div>
   </div>
-  <!-- Prototyping -->
-  <div class="feature gray-background">
+  <!-- Microservices 2.0 -->
+  <div class="feature white-background">
     <div class="container">
       <div class="row">
         <div class="col-sm-6">
-          <h2>Prototyping</h2>
+          <h2>Microservices 2.0</h2>
           <p>
-            Knot.x gives you the ability to use simple Mocks. This allows you to expose your sample
-            data directly to pages even if the real service is not available yet, and switch to
-            the real service without any further development work.
+            Knot.x was build with micro-services principles in mind, all modules are are <b>independent</b>
+            on each other. Each module can run on different data center or host keeping the
+            intercommunication behind the scene. Unified communication between modules over the
+            internal <b>event bus</b> hides the communication complexity and enables you to build each module
+            in languages other than java (<b>ruby</b>, <b>groovy</b>, <b>java script</b>, <b>kotlin</b>, <b>scala</b>).
           </p>
         </div>
         <div class="col-sm-5 col-sm-offset-1">
-          <img src="img/knotx-intro-prototyping-1.png" class="img-responsive img-centered" alt=""/>
+          <img src="img/knotx-microservices.png" class="img-responsive img-centered" alt=""/>
         </div>
       </div>
     </div>
   </div>
-  <!-- Extensions -->
-  <div class="feature white-background">
+  <!-- Scaling -->
+  <div class="feature gray-background">
     <div class="container">
       <div class="row">
         <div class="col-sm-5 col-sm-offset-1">
-          <img src="img/knotx-intro-extensions.png" class="img-responsive img-centered" alt=""/>
+          <img src="img/knotx-scaling.png" class="img-responsive img-centered" alt=""/>
         </div>
         <div class="col-sm-6 text-right">
-          <h2>Extensions</h2>
+          <h2>Scaling</h2>
           <p>
-            Knot.x is a fully modular platform with very flexible extension points that we
-            call Knots and Adapters that efficiently communicates with Knot.x Core using
-            event bus. Thanks to the polyglot nature of Vert.x you can implement extensions
-            in languages other than Java.
+            Scalability is the knot.x first citizen, thanks to the modularity you can scale each
+            module vertically (on one host) and/or horizontally (over multiple hosts) to get best
+            possible performance or response to an unexpected huge load. With message-driven
```

Comment:
_huge load_ sounds super weird, how about `increas in website traffic`?

Reply:
fixed